.aws/task-definition.json

```diff
@@ -21,33 +21,33 @@
       "logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-          "awslogs-group": "awslogs-tis-template",
+          "awslogs-group": "awslogs-tis-revalidation-concerns",
```

Comment:
Make sure that this log group gets created via Terraform in TIS-OPS before the pipeline is uncommented.

Reply:
ok I will